lib/stringify.js

```diff
@@ -75,8 +77,23 @@ var stringify = function stringify(
 ) {
     var obj = object;
 
-    if (sideChannel.has(object)) {
-        throw new RangeError('Cyclic object value');
+    var tmpSc = sideChannel;
+    var step = 0;
+    var findFlag = false;
+    while ((tmpSc = tmpSc.get(sentinel)) !== undefined && !findFlag) {
```

Comment:
is there any way to do this without a loop, or at least without doing an assignment inside the conditional expression?

Reply:
no other graceful way come to mind at the moment, and if I think of any later, I will submit another PR.